.gitignore

```diff
@@ -2,3 +2,16 @@ node_modules
 .idea
 bin/
 npm-debug.log
+
+# OS generated files #
```

Comment:
Oh yes, it will make more sense.
I am still new on Github collaboration and don't have a lot experience.
Thanks for the suggestion @samccone .